src/gui/vue/pages/captureControl/expCapture/organisms/captureToolHeader/RecordButton.vue

```diff
@@ -27,19 +27,20 @@
       :title="$store.getters.message('app.start')"
       id="startButton"
     >
-      <v-icon>fiber_manual_record</v-icon>
+      <v-icon v-if="isDisabled">block</v-icon>
+      <v-icon v-else>fiber_manual_record</v-icon>
     </v-btn>
     <v-btn
       v-else
       icon
       flat
       large
-      color="red"
+      color="green"
```

Comment:
We would like to know why you chose green. (For example, did you think green would be easier to understand, etc.)
If it is simply about your color preferences, then it would be better to make the colors customizable in the settings, because preferences can vary from user to user.

Reply:
I choose green because the users more easily recognize it. I'll try to make it customizable in the settings instead of hard-coded.